doc/cephfs/index.rst

```diff
@@ -57,7 +57,7 @@ for interactive access or scripting via the `cephfs-shell`_.
 .. _deploy MDS manually as needed: add-remove-mds
 .. _Orchestrator deployment table: ../mgr/orchestrator/#current-implementation-status
 .. _Mount CephFS\: Prerequisites: mount-prerequisites
-.. _cephfs-shell: cephfs-shell
+.. _cephfs-shell: ../../man/8/cephfs-shell
```

Comment:
This one still needs to be fixed

Reply:
@varshar16 should be fixed now. thanks for checking the link, Varsha.